src/layouts/Titled/components/TOC.tsx

```diff
@@ -52,10 +52,19 @@ export const TOCBlock: React.FC<{ data: IHeading[] } & HTMLAttributes<HTMLDivEle
 
 const StyledAnchor = styled(Anchor)<{ isCurrent: boolean }>`
   display: block;
+  position: relative;
   margin: ${p => p.theme.space.xxs} 0;
-  margin-left: ${p => p.isCurrent && `-${p.theme.borderWidths.sm}`};
-  border-left: ${p => p.isCurrent && `${p.theme.borders.sm} ${getColor('grey', 800, p.theme)}`};
+  margin-left: ${p => p.theme.space.xxs};
```

Comment:
Should be able to replace shorthand above with...

```suggestion
  margin: ${p => p.theme.space.xxs} 0 ${p => p.theme.space.xxs};
```